maps/CEVEris/_CEV_Eris.dmm

```diff
@@ -60847,10 +60847,15 @@
 /turf/simulated/floor/tiled/steel/techfloor,
 /area/eris/engineering/engine_room)
 "cQR" = (
-/obj/structure/dispenser/oxygen,
 /obj/structure/sign/faction/technomancers{
 	pixel_y = 32
 	},
+/obj/machinery/door/blast/shutters/glass{
+	id = "ineedmorepower";
```

Comment:
Put a normal id onto it so mappers won't get confused

Reply:
Okay